images/win/scripts/Installers/Install-Rust.ps1

```diff
@@ -1,6 +1,7 @@
 ################################################################################
 ##  File:  Install-Rust.ps1
 ##  Desc:  Install Rust for Windows
+##  Supply chain security: checksum validation
```

Comment:
What do you think about mentioning that it is manual validation for the Rustup and controlled by package manager validation for the Rust itself?

Reply:
thank you for the suggestion, much better indeed